pkg/pace.go

```diff
@@ -25,9 +25,12 @@ import (
 )
 
 type pacer struct {
-	phases          []*phase
-	lastRequestTime time.Time
-	done            bool
+	phases                []*phase
+	lastRequestTime       time.Time
```

Comment:
Add comments explaining what `lastRequestTime`, `lastRequestWallTime`, `phaseStartWallTime` and `phaseStartRequestTime` are and what they're used for.